deployments/ansible/molecule/custom_vars/windows-verify.yml

```diff
@@ -211,3 +211,105 @@
     - name: Assert fluentd custom_config.conf is used
       assert:
         that: custom_fluentd_config.stat.checksum == td_agent_config.stat.checksum
+
+    - name: Download signalfx-dotnet-tracing-1.0.0-x64.msi
+      ansible.windows.win_get_url:
+        url: "https://github.com/signalfx/signalfx-dotnet-tracing/releases/download/\
+              v1.0.0/signalfx-dotnet-tracing-1.0.0-x64.msi"
+        dest: "{{ansible_env.TEMP}}"
+      register: dotnet_msi_package
+
+    - name: Install signalfx-dotnet-tracing-1.0.0-x64.msi
+      ansible.windows.win_package:
+        path: "{{dotnet_msi_package.dest}}"
+        state: present
+      check_mode: yes
+      register: msi_installed
+
+    - name: Assert signalfx-dotnet-tracing-1.0.0-x64.msi is already installed
+      assert:
+        that: not msi_installed.changed
+
+    - name: Check SIGNALFX_ENV registry value
+      ansible.windows.win_regedit:
+        path: HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment
+        state: present
+        name: SIGNALFX_ENV
+        data: test-environment
+        type: string
+      check_mode: yes
+      register: reg_value
+
+    - name: Assert registry value exists
+      assert:
+        that: not reg_value.changed
+
+    - name: Check SIGNALFX_SERVICE_NAME registry value
+      ansible.windows.win_regedit:
+        path: HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment
+        state: present
+        name: SIGNALFX_SERVICE_NAME
+        data: test-service-name
+        type: string
+      check_mode: yes
+      register: reg_value
+
+    - name: Assert registry value exists
+      assert:
+        that: not reg_value.changed
+
+    - name: Check SIGNALFX_PROFILER_ENABLED registry value
+      ansible.windows.win_regedit:
+        path: HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment
+        state: present
+        name: SIGNALFX_PROFILER_ENABLED
+        data: "true"
```

Comment:
~~Shouldn't this be configurable by the user? Similar Q to memory profiling.~~

This seems to be a test file.